frontend/app/src/scenes/Home/scenes/UpsertCategory/index.tsx

```diff
@@ -103,6 +133,38 @@ class UpsertCategory extends React.Component<
       });
   };
 
+  addTranslation = () => {
+    this.setState({
+      isAddTranslationSelected: false,
+    });
+  };
+
+  // Checks for the selected language to render the relevant language card
+  handleLanguageSelect = (language: string) => {
+    const clonedTranslations = new Map(this.state.translations);
+    clonedTranslations.set(language, { name: '' });
+    this.setState({
+      isAddTranslationSelected: true,
+      translations: clonedTranslations,
+    });
+  };
+
+  handleTranslations = (languageEnum: string, value: string) => {
+    const settingTranslations = new Map(this.state.translations);
```

Comment:
add a meaningful name with consistency. 

ex line 144 it's `clonedTranslations`

Reply:
sure